src/components/NglViewer/NglViewer.tsx

```diff
@@ -0,0 +1,71 @@
+import { useEffect, useRef } from "react";
+import * as NGL from "ngl";
```

Comment:
For easier tree shaking do not use `* as ...` imports.

```suggestion
import {Stage, type Component} from "ngl";
```